src/compile/config.ts

```diff
@@ -83,32 +83,23 @@ export function orient(mark: Mark, encoding: Encoding, markConfig: MarkConfig =
       }
       // y:Q or Ambiguous case, return horizontal
       return Orient.HORIZONTAL;
+
     case RULE:
-      if (xIsRange) {
-        return Orient.HORIZONTAL;
-      }
-      if (yIsRange) {
-        return Orient.VERTICAL;
-      }
-      if (encoding.y) {
-        return Orient.HORIZONTAL;
-      }
-      if (encoding.x) {
-        return Orient.VERTICAL;
-      }
-      // no x/y -- so it's undefined
-      return undefined;
```

Comment:
will this ever happen? If not, use `never` https://basarat.gitbooks.io/typescript/content/docs/types/never.html